tests/visualization_tests/matplotlib_tests/test_parallel_coordinate.py

```diff
@@ -189,9 +309,33 @@ def test_plot_parallel_coordinate_unique_hyper_param() -> None:
         )
     )
 
-    # both hyperparameters contain unique values
+    # Both hyperparameters contain unique values.
     figure = plot_parallel_coordinate(study_categorical_params)
-    assert len(figure.get_lines()) == 0
+    axes = figure.get_figure().axes
+    assert len(axes) == 3 + 1
+    # Default padding is 5% in Matplotlib.
+    default_padding_fraction = 0.05
+    assert axes[0].get_ylim() == (-default_padding_fraction, default_padding_fraction)
+    assert axes[1].get_ylabel() == "Objective Value"
+    # Optuna's parallel coordinate uses 10% padding for color map.
+    assert axes[1].get_ylim() == (-0.1, 0.1)
+    assert len(figure.findobj(LineCollection)) == 1
+    # Objective values are not sorted by the other parameters,
+    # unlike Plotly's parallel_coordinate.
+    line_collections = figure.findobj(LineCollection)
+    objectives = [line[0, 1] for line in line_collections[0].get_segments()]
```

Comment:
This is still the same logic for fetching objectives, but written slightly different. Could we keep it consistent?

Reply:
Nice catch! I've unified them by defining the same fetch function. Thanks!